distribution/sender/routes/sender-message-consumer.xml

```diff
@@ -0,0 +1,80 @@
+
+        <log message="Received message -> ${body}" />
+        
+        <setProperty name="requestUuid">
+            <jsonpath>$.requestUuid</jsonpath>
+        </setProperty>
+        <setProperty name="messageUuid">
+            <jsonpath>$.messageUuid</jsonpath>
+        </setProperty>
+        
+        <choice>
+            <when>
+                <simple>${exchangeProperty.requestUuid} != null</simple>
+                
+                <setProperty name="requestTableName">
+                    <jsonpath>$.tableName</jsonpath>
+                </setProperty>
+                <setProperty name="requestIdentifier">
+                    <jsonpath>$.identifier</jsonpath>
+                </setProperty>
+                <setProperty name="requestToSave">
+                    <spel>
+                        #{new org.openmrs.eip.app.management.entity.SenderSyncRequest()}
+                    </spel>
+                </setProperty>
+                <script>
+                    <spel>
+                        #{getProperty('requestToSave').setTableName(getProperty('requestTableName'))}
+                        #{getProperty('requestToSave').setIdentifier(getProperty('requestIdentifier'))}
+                        #{getProperty('requestToSave').setRequestUuid(getProperty('requestUuid'))}
+                        #{getProperty('requestToSave').setDateCreated(new java.util.Date())}
+                    </spel>
+                </script>
+                <setBody>
+                    <simple>${exchangeProperty.requestToSave}</simple>
+                </setBody>
+        
+                <log loggingLevel="DEBUG" message="Saving sync request -> ${body}" />
+        
+                <to uri="jpa:SenderSyncRequest" />
+        
+                <log message="Successfully saved sync request" />
+            </when>
+            
+            <when>
+                <simple>${exchangeProperty.messageUuid} != null</simple>
+                
+                <setProperty name="dateSent">
+                    <jsonpath>$.dateSent</jsonpath>
+                </setProperty>
+
+                <setBody>
+                    <spel>
+                        #{new org.openmrs.eip.app.management.entity.SyncResponse(getProperty('messageUuid'), getProperty('dateSent'))}
+                    </spel>
+                </setBody>
+        
+                <log loggingLevel="DEBUG" message="Saving sync response -> ${body}" />
+        
+                <to uri="jpa:SyncResponse" />
+        
+                <log message="Successfully saved sync response" />
+            </when>
+            
+            <otherwise>
+                <!-- TODO Log to a special failures log file or DB -->
+                <log message="Unknowk message was received: ${body}" loggingLevel="ERROR" />
```

Comment:
You have a typo here for Unknown

Reply:
This messages should be logged at warn not error

Reply:
Fixed.